app/views/layouts/application.haml

```diff
@@ -6,6 +6,11 @@
     .noscript-padding
       -# Noscript content will be positioned here
 
+  - if @current_user && @current_community && @current_user.has_admin_rights_in?(@current_community)
+    - with_feature(:onboarding_redesign_v1) do
+      - props = {translations: t('admin.onboarding.topbar'), status: @onboarding_status, guide_root: getting_started_guide_admin_community_path(id: @current_community.id)}
+      - cache ['v1', 'onboarding_topbar', props] do
+        = react_component("OnboardingTopBar", props: props, prerender: true)
```

Comment:
The immediate reason is that the other places are done by Vesa, and this by me :) A better reason is that we haven't agreed on the convention. I think raising is good, I'll add it here